Sources/Fullscreen/TransactionView/TransactionHeaderView/TransactionHeaderView.swift

```diff
@@ -2,4 +2,94 @@
 //  Copyright © 2020 FINN AS. All rights reserved.
 //
 
-import Foundation
+public class TransactionHeaderView: UIView {
+    weak var dataSource: RemoteImageViewDataSource? {
+        didSet {
+            imageView.dataSource = dataSource
+        }
+     }
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .bodyStrong, withAutoLayout: true)
+        label.numberOfLines = 0
+        return label
+     }()
+
+    private lazy var detailLabel: Label = {
+        let label = Label(style: .caption, withAutoLayout: true)
+        label.numberOfLines = 0
+        return label
+    }()
+
+     private lazy var imageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.backgroundColor = .clear
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = (TransactionHeaderView.defaultImageSize / 2)
+        return imageView
+     }()
+
+    private static var defaultImageSize: CGFloat = UIDevice.isIPad() ? 256 : 128
+    private var loadingColor: UIColor? = .toothPaste
+    private var fallbackImage = UIImage(named: .noImage)
+
+    private var model: TransactionHeaderViewModel
+
+    public init(withAutoLayout autoLayout: Bool = false, model: TransactionHeaderViewModel) {
+        self.model = model
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = !autoLayout
+        setup()
+    }
+
+    public func loadImage() {
+        guard let imagePath = model.imagePath else {
+            imageView.image = fallbackImage
+            return
+        }
+
+        let url = "https://images.finncdn.no/dynamic/default/\(imagePath)"
```

Comment:
this looks that can break quite easily, maybe at least use it as a static variable?